github/src/main/kotlin/io/spine/examples/pingh/github/ValuesExts.kt

```diff
@@ -70,3 +73,41 @@ public fun KClass<User>.buildBy(username: String, avatarUrl: String): User =
         Username::class.buildBy(username),
         Url::class.buildBy(avatarUrl)
     )
+
+/**
+ * Creates a new `Mention` with the data specified in the `IssueOrPullRequestFragment`.
+ */
+public fun KClass<Mention>.buildFromFragment(fragment: IssueOrPullRequestFragment): Mention =
+    with(Mention.newBuilder()) {
+        id = NodeId::class.buildBy(fragment.nodeId)
+        author = User::class.buildBy(
+            fragment.whoCreated.username,
+            fragment.whoCreated.avatarUrl
+        )
+        title = fragment.title
+        whenMentioned = Timestamps.parse(fragment.whenCreated)
+        url = Url::class.buildBy(fragment.htmlUrl)
+        vBuild()
+    }
+
+/**
+ * Creates a new `Mention` with the data specified in the `IssueOrPullRequestFragment`.
```

Comment:
This whole idea requires a better description. You refer to `IssueOrPullRequestFragment`, but it's not on he parameter list. You speak of some page title, but you literally require some value to be passed to this method.
It's a mess right now.

Please DO start reading your own PRs prior to requesting a review.